runtime/lua/vim/lsp.lua

```diff
@@ -449,7 +449,10 @@ function lsp.start_client(config)
       --  -- workspace folder in the user interface.
       --  name
       -- }
-      workspaceFolders = nil;
+      workspaceFolders = {{
+        uri=vim.uri_from_fname(config.root_dir);
+        name=string.format("%s", config.root_dir);
```

Comment:
I think we usually have spaces around `=`

Reply:
Nice catch! Fixed!